test/internals/reply.test.js

```diff
@@ -92,161 +41,146 @@ test('reply.serializer should set a custom serializer', t => {
   t.equal(reply._serializer, 'serializer')
 })
 
-test('Reply can set code and header of a response', t => {
-  t.plan(1)
-  try {
-    fastify.get('/auto-status-code', function (req, reply) {
-      reply.send('hello world!')
-    })
-    t.pass()
-  } catch (e) {
-    t.fail()
-  }
-})
+test('within an instance', t => {
+  const fastify = require('../..')()
+  const test = t.test
 
-test('Reply.send should return undefined', t => {
-  t.plan(2)
-  try {
-    fastify.get('/undefined', function (req, reply) {
-      t.strictEqual(reply.send('hello world!'), undefined)
-    })
-    t.pass()
-  } catch (e) {
-    t.fail()
-  }
-})
-
-test('Reply can redirect request', t => {
-  t.plan(1)
-  try {
-    fastify.get('/redirect', function (req, reply) {
-      reply.redirect('/')
-    })
-    t.pass()
-  } catch (e) {
-    t.fail()
-  }
-})
+  fastify.get('/', function (req, reply) {
+    reply.code(200)
+    reply.header('Content-Type', 'text/plain')
```

Comment:
Could you use `type` as below?

Reply:
that's the point of the test, exercising `.header()`.